kubejs/server_scripts/enigmatica/kubejs/base/tags/items/forge/gems.js

```diff
@@ -28,4 +32,9 @@ onEvent('item.tags', (event) => {
     event.get('forge:gems/mana_gem').remove('ars_nouveau:mana_gem');
     event.get('forge:gems/sulfur').add('betterendforge:crystalline_sulphur');
     event.get('forge:gems/amber').add('betterendforge:amber_gem');
+
+    event.add('forge:gems/blazing_crystal', ['powah:crystal_blazing']);

```

Comment:
I think the tag name should be `forge:gems/blazing`, the form it takes is usually left out of the tag - Would that cause any issues? :)